sdk/anomalydetector/Azure.AI.AnomalyDetector/src/Generated/Docs/Multivariate.xml

```diff
@@ -78,7 +78,7 @@ ModelInfo modelInfo = new ModelInfo(new Uri("https://multiadsample.blob.core.win
     {
         AlignMode = AlignMode.Outer,
         FillNAMethod = FillNAMethod.Linear,
-        PaddingValue = 0,
+        PaddingValue = 0F,
```

Comment:
is this expected?

Reply:
yes, it is expected because the type of this property is a `float`.